### ISSUE TITLE
Today widget

### ISSUE THREAD

**Issue by rodericj** [lazerwalker/cortado#13]
Sometimes I drink coffee not at a coffee shop. Should be a pretty simple implementation to have a "yo"-like interface for coffee consumption.

**Comment by lazerwalker** [comment]
Yeah, definitely! Strikes me as the same general idea as #12 – I'd imagine a few buttons for whatever preferred drinks you've set up, as well as an "add other" button that opens the drink selector the same way the option in the notifications do.

I sadly don't have the time to work on anything like this right now (nor, frankly, are sales numbers for the App Store version high enough to be a motivator), but more than happy to merge a PR in if you feel like jamming on it. As you say, should be pretty simple.

**Comment by rodericj** [comment]
Sounds great. I'd love to have assigned it to myself. I'll put it on the list of things to do. No timeframe though.

**Comment by rodericj** [comment]
I hadn't seen #12. That is actually really easy as well

**Comment by rodericj** [comment]
I found out you can do exactly this with a workflow app today widget. That discovery is going to downgrade this priority for me. Possibly to 0.

**Comment by rodericj** [comment]
#12 can also be done with workflow.
